setup_wizard/Dockerfile

```diff
@@ -1,5 +1,7 @@
-FROM python:3
+FROM python:3.7
 WORKDIR /app
 ADD . /app
 RUN pip install -r requirements.txt
+LABEL maintainer="t@a4.io"
+LABEL pub.microblog.oneshot=true
```

Comment:
Is this kind of label a "standar", or it's just informative? I have not been able found any information.

Reply:
Yeah maintainer is best practices, the other is namespaced and just descriptive.